subprojects/docs/src/docs/userguide/developing-plugins/publishing_gradle_plugins.adoc

```diff
@@ -184,8 +185,41 @@ If you browse the associated page on the {portal} for the https://plugins.gradle
 .GradleTest plugin metadata on the {portal}
 image::plugin-portal-plugin-page.png[]
 
+[[custom-plugin-repositories]]
+== Publish your plugin to a local repository
 
-== Publish your plugin
+To check how the artifacts of your published plugin look, or to use it only locally or internal in your company, you can publish it to any maven repository, including a local folder.
+For that, you only need to <<publishing_maven.adoc#publishing_maven:repositories,configure repositories for publishing>>.
+Then you can run the `publish` task to publish your plugin to all repositories you have defined.
+
+.build.gradle
+[source.multi-language-sample,groovy]
+----
```

Comment:
This is an inline snippets, because all snippets in this chapter are inline already. I'll move them all into one snippet build in a follow up.